app/views/reports/regions/show.html.erb

```diff
@@ -228,6 +228,55 @@
   <%= render(Dashboard::Hypertension::LostToFollowUpComponent.new(data: @data, region: @region, period: @period)) %>
 </div>
 
+<% if Flipper.enabled?(:overdue_patients, current_admin) %>
+  <div class="d-flex jc-space-between">
+    <h4 class="mt-5 mb-32px">Overdue</h4>
+
+    <div class="d-flex" style="margin-top: 43px;">
+      <form method="get" name="overdue_form">
+        <div class="custom-control custom-switch checkbox-outside-card">
+        <input type="checkbox"
+             onChange="this.form.submit()"
+             name="with_non_contactable"
+             for="overdue_form"
+             class="custom-control-input"
+             id="overdue-section"
+             <% if @with_non_contactable %>checked<% end %>>
+        <label class="font-weight-normal custom-control-label <% unless false %>c-grey-dark<% end %>"
+             for="overdue-section">
+             Include non-contactable patients
+        </label>
+        </div>
```

Comment:
Let's extract this into a separate component like the ltfu toggle. Not a blocker for release though.